org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Play videos from the net
+  - --share=network
+  # OpenGL rendering
+  - --device=dri 
+  # Webcam capturing
+  - --device=all
+  # Audio output
+  - --socket=pulseaudio
+  # Play video files, but only read access required
+  - --filesystem=host:ro
+cleanup:
+  - '/include'
+  - '/lib/*.a'
+  - '/lib/cmake'
+  - '/lib/debug'
+modules:
+  - name: openvr
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/ValveSoftware/openvr/archive/refs/tags/v1.14.15.tar.gz
+        sha256: 40644e569f057e86c733d818a22d821c640307e5058d95958414422134d79584
+    cleanup:
+      - '/share/pkgconfig'
+  - name: vrpn
+    buildsystem: simple
```

Comment:
why no using the `cmake-ninja` buildsystem?

Reply:
I could not get it to work. VRPN's use of CMake is ... hard to deal with. They require an extra build directory, and passing the -B option does not work. I found no better way to build VRPN.

Reply:
But at least I switched the generator to ninja.

Reply:
Ah - thanks, I did not know about that, I only looked in the manifest documentation and not the flatpak-builder documentation. Fixed now and works with `cmake-ninja`.